### ISSUE TITLE
Serving Mleap Serialized 

### ISSUE THREAD

**Issue by HassanRaza1313** [mlflow/mlflow#7530]

**Comment by WeichenXu123** [comment]
Yes currently mlflow does not support saving MLeap model as pyfunc model. Are you interested in contribute this feature ?

**Comment by dbczumar** [comment]
Hi @HassanRaza1313, when we first introduced the MLeap flavor, MLeap did not support scoring / serving in Python. We're happy to extend the MLflow MLeap to support pyfunc if MLeap has introduced APIs for loading and scoring MLeap models in Python; please let us know if this is the case.

**Comment by Usmaniatech** [comment]
Does mlpeap support serving in pyspark ?

**Comment by WeichenXu123** [comment]
From my understanding, the mleap added interface `Transformer.deserializeFromBundle` will return a pyspark transformer (or model) loaded from saved mleap model. The returned object we can run predict method like other pyspark models. CC @dbczumar @Usmaniatech

**Comment by B-Step62** [comment]
The `mlflow.mleap` flavor has since been removed from MLflow (MLeap serialization is no longer supported), so serving MLeap-serialized models via MLflow is not available. Closing as obsolete; use the standard `mlflow.spark` flavor for SparkML models.
